src/radius/nss/libnss-radius/nss_radius_common.c

```diff
@@ -159,11 +159,11 @@ static void init_rnm(RADIUS_NSS_CONF_B * conf) {
     rnm[0].gid   = 999;
     rnm[0].groups = "docker";
```

Comment:
The readonly user should be in `users` group, and no `docker` group membership.

Reply:
Hi @qiluo-msft. User groups were defined based on existing RADIUS design document of SoNiC [RADIUS_Design](https://github.com/sonic-net/SONiC/blob/master/doc/aaa/radius_authentication.md). I haven't introduced them as part of this pull request. Can you please confirm if we have to modify the same ?